baremetal/metal3machine_manager_test.go

```diff
@@ -4761,22 +4778,30 @@ func newBareMetalHost(name string,
 	status *bmov1alpha1.BareMetalHostStatus,
 	powerOn bool,
 	autoCleanMode string,
-	clusterlabel bool) *bmov1alpha1.BareMetalHost {
+	clusterlabel bool,
+	bmhUID string) *bmov1alpha1.BareMetalHost {
 	if name == "" {
 		return &bmov1alpha1.BareMetalHost{
 			ObjectMeta: testObjectMeta("", namespaceName, ""),
 		}
 	}
 
+	BMHUid := Bmhuid
```

Comment:
I think it would be better to keep this lower case to avoid confusion with exported variables. Perhaps we could change it to `uid` to keep things simple?
```suggestion
	uid := Bmhuid
```